frontend/src/scenes/settings/environment/ChannelType.tsx

```diff
@@ -0,0 +1,48 @@
+import { useActions, useValues } from 'kea'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
+import { useState } from 'react'
+import { teamLogic } from 'scenes/teamLogic'
+
+import { CustomChannelRule } from '~/queries/schema'
+import { LemonInput } from 'lib/lemon-ui/LemonInput'
+
+export function ChannelType(): JSX.Element {
```

Comment:
This code is placeholder and behind a FF, so it's ok that's imperfect. It only exists to split this PR into 2 rather 1 big PR